include/z64save.h

```diff
@@ -510,10 +508,36 @@ typedef enum {
 #define SET_STOLEN_ITEM_2(itemId) \
     (gSaveContext.save.saveInfo.stolenItems = (gSaveContext.save.saveInfo.stolenItems & ~0x00FF0000) | ((itemId & 0xFF) << 0x10))
 
-#define GET_TOWN_SHOOTING_GALLERY_HIGH_SCORE() ((s32)(gSaveContext.save.saveInfo.shootingGalleryHighScores & 0xFFFF))
-#define GET_SWAMP_SHOOTING_GALLERY_HIGH_SCORE() ((s32)((gSaveContext.save.saveInfo.shootingGalleryHighScores & 0xFFFF0000) >> 0x10))
-#define SET_TOWN_SHOOTING_GALLERY_HIGH_SCORE(score) (gSaveContext.save.saveInfo.shootingGalleryHighScores = (gSaveContext.save.saveInfo.shootingGalleryHighScores & 0xFFFF0000) | ((u16)(score)))
-#define SET_SWAMP_SHOOTING_GALLERY_HIGH_SCORE(score) (gSaveContext.save.saveInfo.shootingGalleryHighScores = ((gSaveContext.save.saveInfo.shootingGalleryHighScores) & 0xFFFF) | ((u16)(score) << 0x10))
+typedef enum HighScore {
+    /* 0 */ HS_BANK_RUPPEES,
+    /* 1 */ HS_UNK_1,
+    /* 2 */ HS_FISHING,
+    /* 3 */ HS_BOAT_ARCHERY,
+    /* 4 */ HS_HOSRE_BACK_BALLOON,
+    /* 5 */ HS_LOTTERY_GUESS,
+    /* 6 */ HS_SHOOTING_GALLERY
+} HighScore;
+
+#define HIGH_SCORE(type) (gSaveContext.save.saveInfo.highScores[(type)])
+
+#define HS_GET_BANK_RUPPEES() (HIGH_SCORE(HS_BANK_RUPPEES) & 0xFFFF)
+#define HS_SET_BANK_RUPPEES(ruppees) (HIGH_SCORE(HS_BANK_RUPPEES) = ((HIGH_SCORE(HS_BANK_RUPPEES) & 0xFFFF0000) | (ruppees)))
```

Comment:
```suggestion
#define HS_GET_BANK_RUPEES() (HIGH_SCORE(HS_BANK_RUPEES) & 0xFFFF)
#define HS_SET_BANK_RUPEES(rupees) (HIGH_SCORE(HS_BANK_RUPEES) = ((HIGH_SCORE(HS_BANK_RUPEES) & 0xFFFF0000) | (rupees)))
```
Rupees has 1 p, not 2. Applies to a few places